python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -373,8 +385,10 @@ class RandomForestRegressor(Base):
             <RandomForestMetaData[double, double]*>params_t64
 
         state['verbose'] = self.verbose
+        state["file_name"] = self.file_name
+        state["model_pbuf_bytes"] = self.model_pbuf_bytes
```

Comment:
why is the temporary file name needed to be stored in state, if you are storing the actual bytes? Wouldn't it be better to just remove the temp file and leave the bytes in state?